modules/administration-guide/pages/creating-a-telemetry-plugin.adoc

```diff
@@ -333,14 +333,14 @@ include::example$creating-a-telemetry-plug-in/plugin.yaml[]
 
 Typically, the user deploys this file to a corporate web server. This guide demonstrates how to create an Apache web server on OpenShift and host the plug-in there.
 
-Create a ConfigMap referencing the new `plugin.yaml` file.
+Create a config map referencing the new `plugin.yaml` file.
```

Comment:
To be honest I have some doubt, since we create a `ConfigMap` object but not `config map`

Reply:
Ironically, I didn't want to touch it. That's been made into a Vale rule, and Vale now found even more instances of `ConfigMap` that it treats as a an error:
https://github.com/eclipse-che/che-docs/actions/runs/3160023292/jobs/5143981433#step:6:27
Let me think.

Reply:
I've looked up the supplementary style guide, and now my understanding is the Vale rule is an error:
![image](https://user-images.githubusercontent.com/78564304/193530938-91c107c7-d4ff-4081-a3c4-b12e1212fb46.png)

Reply:
@tolusha, thank you for pointing it out. I will open an issue in the Vale repository. That should not have been made into an error.

Reply:
Vale issue opened: https://github.com/redhat-documentation/vale-at-red-hat/issues/326